### ISSUE TITLE
Oceanfront: Screen Reader - Add the missing items and fix issues from the list

### ISSUE THREAD

**Issue by alexivanenko** [1CRM/oceanfront#437]
Color Picker:
DateTime picker:
- [x] not all elements available 

OfField:
- [x] fix tab index for prepend icon

Toggle Inputs:
- [x] focus state not visible
- [x] add label

Radio Input:
Slider: 
HTML Editor: 
- [x] add labels

DataTable:
- [x] no way to navigate inside the editable table

OfDialog
- [x] add tab index for the close button [X]
- [x] [X] not clickable from the keybord

File Input:
- [x] add tab index for the clear button [X]

MultiSelect:
- [x] no way to navigate inside the popup
